.github/workflows/test.yaml

```diff
@@ -22,6 +22,11 @@ jobs:
     - name: Run Build
       run: mvn -DskipTests clean package
     - name: Run Tests
+      env:
+        OKTA_DUMMY_CI_PW: ${{ secrets.OKTA_DUMMY_CI_PW }}
+        WCS_DUMMY_CI_PW: ${{ secrets.WCS_DUMMY_CI_PW }}
+        OKTA_CLIENT_SECRET: ${{ secrets.OKTA_CLIENT_SECRET }}
+        AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
       run: |
         docker-compose -f src/test/resources/docker-compose-test.yaml pull
         mvn clean test
```

Comment:
This won't work for forks. What we could do is to run the tests with the env variables for internal PRs and without for externals ([python example](https://github.com/semi-technologies/weaviate-python-client/pull/207/files))

Reply:
the docker-compose pull won't work for forks? or `mvn clean test`?

Reply:
The secrets are empty. For forks it tries to login with an empty password and the authentication tests fail with "wrong credentials". If the environment variables are not defined for forks, the tests will just be skipped

Reply:
Tests will be skipped if those secrets will be empty

Reply:
forks builds will be run but with OIDC tests being skipped

Reply:
I'm not sure if that is true, I had to change it in python. Could you double-check?

Reply:
I will merge this, create a PR from my fork and test if that works, but I'm pretty sure it will